guides/common/modules/proc_creating-custom-snippets-to-apply-errata.adoc

```diff
@@ -0,0 +1,35 @@
+[id="creating-custom-snippets-to-apply-errata_{context}"]
+= Creating custom snippets to apply errata
```

Comment:
Same comments as above. Perhaps this: "Running custom code while applying errata"